glue_jupyter/table/table.vue

```diff
@@ -15,7 +15,7 @@
       <template v-slot:header="props">
         <thead>
           <tr>
-            <th style="padding: 0 10px; width: 40px">#</th>
+            <th :style="'padding: 0 10px; width: '+Math.ceil(Math.log10(total_length))*20+'px'">#</th>
```

Comment:
Quite a bit of Math here! I am well out of my depth with the syntax, but I assume there is no equivalent of
`len(f"{total_length}")`? 😆

Reply:
`total_length` is an integer of how many entries are in the table.  So `ceil(log10(total_length))` is intended to find the maximum number of digits needed, by mapping 1 to 0 digits (perhaps we should have an exception to force this to 1?), 2-10 to 1 digit, 11-100 to 2 digits, etc, which with the exception of `total_length==1` is what would be expected since the table is zero-indexed.

Reply:
Yes, it's clear that the method works, I was just a little lost in the remaining vuetify template code around it.
It might certainly be done the Python way as well, by adding a `max_width` to `TableBase`, but maybe not worth it as it would need to be updated as well.